test/kb_orthofinder_server_test.py

```diff
@@ -121,21 +121,23 @@ def test_annotate_plant_transcripts(self):
                         'input_genome' : input_genome,
                         'threshold' : 0.55}
 
-        if(self.cfg['testing'] == 1):
+        if(self.cfg['testing'] == '1'):
```

Comment:
brackets

Checking for a `testing` config variable in a test file seems a bit counterintuitive... why would the test file be executed if not because tests were being run?

Reply:
`testing` is a bit of a misnomer, the full reference data means that it has to run on 20+ plant genomes and takes several hours, so by `testing` I mean I'm testing locally with a much smaller number of genomes, and it doesn't take so long to iterate on.

Reply:
If that isn't documented anywhere, there should be a comment added so that other devs who work on this code will also know what this means.

Reply:
Changed `testing` flag to `skip_refdata` flag to better reflect what's going on in 9ca841d60089414785b076375ae088febb9d0668 and 9ca841d60089414785b076375ae088febb9d0668